src/OVAL/oval_generator.c

```diff
@@ -114,6 +117,16 @@ char *oval_generator_get_timestamp(struct oval_generator *generator)
 	return generator->timestamp;
 }
 
+const char *oval_generator_get_platform_schema_version (struct oval_generator *generator, const char *platform)
+{
+	char *platform_schema_version = oscap_htable_get(generator->platform_schema_versions, platform);
```

Comment:
I don't understand the logic here. The platform version is implicitly the core version if it was not specified in the OVAL file? Can you quote the spec about this?